File: src/GenerateDocstring/dart/GenerateDocstring.dart
/// An annotator for generating docstrings for dart
/// classes, enums, and functions
class GenerateDocstring {
  final String description;
  final String? codeExample;
  final String annotatorName;

  const GenerateDocstring({
    required this.description,
    this.codeExample,
    this.annotatorName = "GenerateDocstring",
  });
}

File: src/GenerateDocstring/java/GenerateDocstring.java
package GenerateDocstring.java;
/// An annotator for generating docstrings for java
/// classes, enums, and functions 
public class GenerateDocstring {
    String description;
    String codeExample = null;
    String annotatorName;

    public GenerateDocstring(String description, String codeExample) {
        this.description = description;
        this.codeExample = codeExample;
        this.annotatorName = "GenerateDocstring";
    }
}